### DISCUSSION TITLE
I hope to increase the CPU usage of code-server

### DISCUSSION THREAD

**Discussion by LeiyuanBlog** [coder/code-server#3995] [Q&A]
I was running on an 8C16G server, but I found that it only took one core and 2GB of memory.This has resulted in slow response to code prompts when I write Python.

![截屏2021-08-16 下午1 50 19](https://user-images.githubusercontent.com/44362360/129517096-14f108fa-ceaa-4ace-927e-b37659d91973.png)

**Comment by jsjoeio** [comment]
Hmm I don't know how we can help but maybe @code-asher does

**Reply by lg-nineoneninethreenine** [reply]
Well, if you increase the CPU usage of code-server, it is danger levels. For the computers having 4 GB RAM or less, it would just slow down the entire computer. If the memory chips are sort of 8 GB or 16 GB, the computer can manage. But, the computers having 4 GB or less will have a very slower output.

**Comment by code-asher** [comment]
I don't believe it's possible since JavaScript is single-threaded. The extension host will only use one core.

If the prompts are slow could it be due to network speeds?

**Reply by code-asher** [reply]
Ah nevermind I see the core is maxed out. That seems to be Python itself and not code-server so I'm not sure what can be done there if anything. You might want to search for help about Python taking 100% CPU.
